tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/TeamUserGenerator.cs

```diff
@@ -0,0 +1,133 @@
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <param name="team">Octokit.Team to get users for.</param>
+        public static async Task GetUsersForTeam(GitHubEventClient gitHubEventClient, Team team)
+        {
+            // If this team has already been added to the dictionary then there's nothing to do. This
+            // should prevent any weirdness if there ends up being some kind of circular team reference
+            if (_teamUserDict.ContainsKey(team.Name))
+            {
+                return;
+            }
+            // Get all of the team members
+            var teamMembers = await gitHubEventClient.GetTeamMembers(team);
+            if (teamMembers.Count > 0)
+            {
+                // Just need a List<string> containing the logins from the returned
+                // list of users. The Login is what's used in @mentions, assignments etc
+                var members = teamMembers.Select(s => s.Login).ToList();
+                _teamUserDict.Add(team.Name, members);
+            }
+            else
+            {
+                // It seems better to report this than to add a team to the dictionary with no users
+                Console.WriteLine($"Warning: team {team.Name} has no members and will not be added to the dictionary.");
+            }
+            var childTeams = await gitHubEventClient.GetAllChildTeams(team);
+            foreach (Team childTeam in childTeams)
+            {
+                await GetUsersForTeam(gitHubEventClient, childTeam);
+            }
+        }
+
+        /// <summary>
+        /// This method is called after the team/user data is stored in blob storage. It verifies that the
+        /// team/user data from blob storage is the same as the in-memory data that was used to create the blob.
+        /// </summary>
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <returns>True, if the team data in storage matches the in-memory data that was used to create the blob otherwise false.</returns>
+        public static async Task<bool> VerifyStoredTeamUsers(GitHubEventClient gitHubEventClient)
+        {
+            bool hasError = false;
+            string rawJson = await gitHubEventClient.GetTeamUserBlobFromStorage();
+            var list = JsonSerializer.Deserialize<List<KeyValuePair<string, List<string>>>>(rawJson);
+            var storedDictionary = list.ToDictionary((keyItem) => keyItem.Key, (valueItem) => valueItem.Value);
+
+            // Verify the dictionary from storage contains everything from the _teamUserDict
+            if (_teamUserDict.Keys.Count != storedDictionary.Keys.Count)
+            {
+                // At this point list the teams and return, don't bother looking at the users.
+                Console.WriteLine($"Error::Created dictionary has {_teamUserDict.Keys.Count} teams and stored dictionary has {storedDictionary.Keys.Count} teams.");
```

Comment:
```suggestion
                Console.WriteLine($"Error: Created dictionary has {_teamUserDict.Keys.Count} teams and stored dictionary has {storedDictionary.Keys.Count} teams.");
```

Reply:
Done. Will be pushed in the next commit. In the future, please make the suggestion and I'll do things locally. I don't like to commit suggestions, I'd rather make the change locally and add a commit.